modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Configuring mount namespace encapsulation
+
+You can configure mount namespace encapsulation so that a cluster runs with less resource overhead.
```

Comment:
```suggestion
You can configure mount namespace encapsulation so that a cluster runs with less resource overhead.
While mount namespace encapsulation is a Technology Preview feature, it is disabled by default, and must be manually enabled.
```